include/equalizer_ui.hpp

```diff
@@ -56,7 +79,15 @@ class EqualizerUi : public Gtk::Grid, public PluginUiBase {
 
   void on_import_apo_preset_clicked();
 
+  bool parse_apo_filter(const std::string& line, struct ImportedBand& filter);
+
   void import_apo_preset(const std::string& file_path);
+
+  std::unordered_map<std::string, FilterType> const FilterTypeMap = {
```

Comment:
This is ugly but gets the job done. If anyone has a better idea on how to better parse this godforsaken spec, I'll gladly accept it.